integration-tests/testutils/common/common.go

```diff
@@ -118,13 +119,20 @@ func (s *SnappySuite) SetUpTest(c *check.C) {
 	}
 }
 
+// OSSnapName returns the name of the OS snap.
+func OSSnapName(c *check.C) string {
+	snappyOS, err := partition.SnappyOS()
+	c.Assert(err, check.IsNil, check.Commentf("Error getting the name of the OS snap: %s", err))
+	return strings.Split(snappyOS, ".")[0]
+}
+
```

Comment:
partition is clearly not a good place for this. But the right place depends on how https://bugs.launchpad.net/snappy/+bug/1532245 is fixed.
For now I'll put it in partition, which we also have to rename. And when the bug is fixed, we'll have to update this and it will be easier to find a name for that new package.